src/views/name.njk

```diff
@@ -10,12 +10,12 @@
       
       {% include "includes/csrf_token.njk" %}
 
-      <h1 class="govuk-heading-xl">Limited partnership name</h1>
+      <h1 class="govuk-heading-xl">{{ i18n.namePage.title }}</h1>
```

Comment:
Where is the i18n value set? Also how?

Reply:
It's a lengthy response to have replied via Slack